OmarEspinosaCastro/30-Agosto/ArrayList.java

```diff
@@ -0,0 +1,29 @@
+
+package collection;
```

Comment:
How is this works without imports?!!! Unbelievable!
I don't think you have test this right?

Reply:
I run the code, only to remove the components and imports
![Sin título](https://user-images.githubusercontent.com/38709814/131901511-2b049041-d4a2-4508-a12c-fa0752a3e3fd.png)